stubs/sublime.pyi

```diff
@@ -633,7 +633,7 @@ class View:
         ...
 
     def find_all(self, pattern: str, flags: int = ..., fmt: Optional[Any] = ...,
-                 extractions: Optional[Any] = ...) -> 'List[Region]':
+                 extractions: Optional[Any] = ...) -> 'Iterable[Tuple[int, int]]':
```

Comment:
Another example where ViewLike and sublime.View were incompatible. Had to change the signature here.